app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -62,100 +62,85 @@ You’ll earn a [competitive salary](/salaries-and-benefits), get supported in y
-
-You may also be eligible for another type of visa, or you may have a pre-existing right to work in the UK, such as settled or pre-settled status under the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain in the UK](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk).
-
-[If you're an Irish citizen you do not need a visa](https://www.gov.uk/government/publications/common-travel-area-guidance).
-
-## Consider getting qualified teacher status (QTS)
-
-[Qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) is the professional status teachers in England gain at the end of their teacher training.
+[Qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) is the professional status teachers in England gain at the end of their teacher training. 
 
 If you’re a qualified teacher from outside the UK, you can [work as a teacher in England for up to 4 years without QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#qts-exemption-for-teachers-from-outside-the-uk).
 
 After that, you will need QTS to teach in many schools in England.
 
-It’s not a legal requirement in all types of schools (for example, [academy schools, free schools and private schools](https://www.gov.uk/types-of-school)).
-
-However, even in these schools, having QTS can help your application for a teaching job.
-
-## The different routes to QTS
-
-If you decide QTS is right for you, the most appropriate route will depend on the country you’re recognised in, your qualifications and your teaching experience.
+QTS is not a legal requirement in some types of school (for example, [academy schools, free schools and private schools](https://www.gov.uk/types-of-school)). However, the majority of schools use QTS as a measure of teaching skills and experience, so you will find it much easier to get a teaching job in England if you have QTS.
 
-### Apply directly to England’s Teaching Regulation Agency (TRA)
+It’s important to note that, even if you decide to apply for QTS, you will have to apply for your teaching job and your visa separately. 
 
-You can apply for QTS directly to the TRA if you meet the following 3 requirements:
+### How to apply for QTS
 
-1. You have a teaching qualification from:
+From 1 February 2023, if you have a teaching qualification from the following countries and regions, you will be able to access a new service, called [Apply for qualified teacher status in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start), to check if you meet the requirements for QTS:
```

Comment:
Suggestion: 
From 1 February 2023, if you have a teaching qualification from the following countries and regions, you'll be able to apply to check if you meet the requirements for QTS: